app/Party.php

```diff
@@ -394,7 +401,15 @@ public function scopeUpcomingEvents($query, $by_event = false)
 
       return $this->join('groups', 'groups.idgroups', '=', 'events.group')
             ->join('users_groups', 'users_groups.group', '=', 'groups.idgroups')
-            ->whereNotNull('events.wordpress_post_id')
+            ->leftJoin('events_users', function($join) {
```

Comment:
We're joining to events for a specific user here, but not in the $by_event path above - any reason?

Reply:
Will discuss above.